### ISSUE TITLE
is this still functional?

### ISSUE THREAD

**Issue by neo-neo1** [interlark/123tv-iptv#11]
Could someone please confirm if this is still working? As simple as that.

I know ustvgo was shutdown and my 123tv install is currently down as well. I installed this a while ago but haven't used it in a while until now.

**Comment by Nostradamus1973** [comment]
Yes, It's down and most likely staying down, a simple way to check is to visit their site, you'll get something like a 404 error if you go there.

**Comment by interlark** [comment]
Thanks for the info, looks like [123tv](123tv.live) is indeed down.
Man, idk whats happening with all these IPTV providers... first ustvgo, now this.

**Comment by Nostradamus1973** [comment]
> Thanks for the info, looks like [123tv](123tv.live) is indeed down. Man, idk whats happening with all these IPTV providers... first ustvgo, now this.

Hi Andy,
I hope you're doing well.
Do we(you) have any viable replacements for 123tv/ustvgo?

Thx and have a great day.

**Comment by interlark** [comment]
Hi, @Nostradamus1973.
@s66jones from [ustvgo issues](https://github.com/interlark/ustvgo-iptv/issues/62) gave me a [hint](https://github.com/interlark/ustvgo-iptv/issues/62#issuecomment-1411408012) about [daddyhd](https://daddylivehd.sx/) website, I think I could make a port for it.

**Comment by Nostradamus1973** [comment]
> Hi, @Nostradamus1973. @s66jones from [ustvgo issues](https://github.com/interlark/ustvgo-iptv/issues/62) gave me a [hint](https://github.com/interlark/ustvgo-iptv/issues/62#issuecomment-1411408012) about [daddyhd](https://daddylivehd.sx/) website, I think I could make a port for it.

That would be amazing if you could do that Andy, let me know when you're ready for us to test it out.
Thx

**Comment by ramirous** [comment]
> @s66jones from [ustvgo issues](https://github.com/interlark/ustvgo-iptv/issues/62) gave me a [hint](https://github.com/interlark/ustvgo-iptv/issues/62#issuecomment-1411408012) about [daddyhd](https://daddylivehd.sx/) website, I think I could make a port for it.

Thank you! Your ustvgo port worked wonderfully while it lasted, thank you for your work =)

**Comment by skiely** [comment]
> Hi, @Nostradamus1973.
> @s66jones from [ustvgo issues](https://github.com/interlark/ustvgo-iptv/issues/62) gave me a [hint](https://github.com/interlark/ustvgo-iptv/issues/62#issuecomment-1411408012) about [daddyhd](https://daddylivehd.sx/) website, I think I could make a port for it.

That would be great 😃

**Comment by Nostradamus1973** [comment]
@interlark 
Hi Andy, I hope you're doing well.
I wanted to check in with you.
Have you made any progress with the daddyhd website?

Thx Andy, have a great day, ttyl

**Comment by Nostradamus1973** [comment]
Good morning Everyone,
Has anyone heard of this site:
[https://www.distro.tv/](https://www.distro.tv/) ?
I'm not sure if it can be ported or not.
I didn't want to bother Andy if the question was already asked.
Thx and have a great day.
